lightningd/peer_control.c

```diff
@@ -2333,7 +2333,7 @@ static struct command_result *json_listpeers(struct command *cmd,
 	if (!param(cmd, buffer, params,
 		   p_opt("id", param_node_id, &specific_id),
 		   p_opt("level", param_loglevel, &ll),
-		   p_opt("query", param_string, &querystr),
+		   p_opt("query", param_escaped_string, &querystr),
```

Comment:
This has also since changed. Now we have a separate "graphql" method. My only hesitation with conditional-compiling (the "graphql" method) would be that we wouldn't be able to take advantage of any of it in other parts of our non-experimental code, but I would certainly defer to others on that.